onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -377,6 +377,27 @@ ONNX_MS_OPERATOR_SET_SCHEMA(FastGelu, 1,
                                   return true;
                                 }));
 
+ONNX_MS_OPERATOR_SET_SCHEMA(RelPosAttnBiasGen, 1,
+                            OpSchema()
+                                .SetDoc("Compute binned relative position bias")
+                                .Attr("max_distance", "Max distance", AttributeProto::INT)
+                                .Attr("is_bidirectional", "Default value is 0.", AttributeProto::INT, static_cast<int64_t>(0))
+                                .Input(0, "bias_table", "2D input tensor with shape (num_buckets, num_heads), COL-major(See UT for example)", "T")
+                                .Input(1, "sequence_length", "Real sequence length from last dimension of hidden states", "U")
+                                .Output(0, "output", "4D output tensor with shape (1, num_heads, sequence_length, sequence_length)", "T")
+                                .TypeConstraint("T", {"tensor(float)", "tensor(float16)"}, "Constrain input and output types to float or half tensors.")
+                                .TypeConstraint("U", {"tensor(int64)"}, "Constrain sequence_length to int tensors.")
+                                .TypeAndShapeInferenceFunction([](ONNX_NAMESPACE::InferenceContext& ctx) {
+                                  propagateElemTypeFromInputToOutput(ctx, 0, 0);
+                                  auto& bias_table_shape = getInputShape(ctx, 0);
+                                  TensorShapeProto output_shape;
+                                  output_shape.add_dim()->set_dim_value(1);
+                                  *output_shape.add_dim() = bias_table_shape.dim(1);
+                                  output_shape.add_dim();
+                                  output_shape.add_dim();
+                                  updateOutputShape(ctx, 0, output_shape);
```

Comment:
It is better also change symbolic shape inference script for shape inference.

Reply:
not sure if it's necessary since the output shape is determined runtime, the "auto_merge" mode in symbolic shape infer will be used either way.

Reply:
You can postpone it to another PR. I guess we will encounter issue in shape inference in convert_generation when there is such node.

Reply:
> You can postpone it to another PR. I guess we will encounter issue in shape inference in convert_generation when there is such node.

sure, will add with the fusion script if needed